CHANGELOG.md

```diff
@@ -396,35 +394,76 @@ Added TLS support
+* Agent [Core][agent-core] that provides plugin lifecycle management
 (initialization and graceful shutdown of plugins) is able to run
-different [flavors](../v1.0.2/flavors) (reusable collection of plugins):
-    * [local flavor](../v1.0.2/flavors/local) - a minimal collection of plugins:
-      * [statuscheck](health/statuscheck)
-      * [servicelabel](servicelabel)
-      * [resync orch](datasync/restsync)
-      * [log registry](logging)
-    * [RPC flavor](../v1.0.2/flavors/rpc) - exposes REST API for all plugins, especially for:
-      * [statuscheck](health/statuscheck) (RPCs probed from systems such as K8s)
-      * [logging](logging/logmanager) (for changing log level at runtime remotely)
+different flavors (removed in v1.5) (reusable collection of plugins):
+    * local flavor - a minimal collection of plugins:
+      * [statuscheck][statuscheck-plugin]
+      * [servicelabel][service-label-plugin]
+      * [resync orch]([datasync-restsync]
+      * [log registry][log-registry]
+    * RPC flavor - exposes REST API for all plugins, especially for:
+      * [statuscheck][statuscheck-plugin] (RPCs probed from systems such as K8s)
+      * [logging][logmanager-plugin] (for changing log level at runtime remotely)
     * connector flavors:
       * Cassandra flavor
       * etcd flavor
       * Redis flavor
       * Kafka flavor
-* [Examples](examples)
-* [Docker](docker) container-based development environment
+* [Examples][examples]:
+* [Docker][docker] container-based development environment
 * Helpers:
-  * [IDX Map](idxmap) is a reusable thread-safe in memory data structure.
+  * [IDX Map][index-map] is a reusable thread-safe in memory data structure.
       This map is designed for sharing key value based data
       (lookup by primary & secondary indexes, plus watching individual changes).
       It is useful for:
       - implementing backend for plugin specific API shared by multiple plugins;
       - caching of a remote data store.
-  * [Config](config): helpers for loading plugin specific configuration.
-  * [Service Label](servicelabel): retrieval of a unique identifier for a CN-Infra based app.
+  * [Config]([config]: helpers for loading plugin specific configuration.
+  * [Service Label][service-label-plugin]: retrieval of a unique identifier for a CN-Infra based app.
+  
+[agent-core]: agent 
+[bolt-plugin]: db/keyval/bolt
+[cassandra-plugin]: db/sql/cassandra
+[config]: config
+[consul-plugin]: db/keyval/consul
+[cryptodata-plugin]: db/cryptodata
+[datasync-api]: datasync/datasync_api.go
+[datasync-plugin]: datasync
+[datasync-restsync]: datasync/restsync
+[docker]: docker
+[docs-filedb]: http://docs.ligato.io/en/latest/plugins/db-plugins/#filedb
```

Comment:
I would probably avoid linking to docs from CHANGELOG and rather update the main [README](https://github.com/ligato/cn-infra/blob/dev/README.md) to link to docs instead. 

Also please update `## Documentation` section in README to link to docs too.